packages/jss/tests/functional/sheet.js

```diff
@@ -515,6 +515,22 @@ describe('Functional: sheet', () => {
     })
   })
 
+  describe('sheet.update()', () => {
```

Comment:
Please also add a test where the property becomes `null` and `false`

Reply:
will do